apps/time/src/widgets/sugang/ui/SugangLayout.tsx

```diff
@@ -0,0 +1,92 @@
+'use client';
+
+import { Dispatch, useState } from 'react';
+
+import { KguLogoColor } from '@clab-platforms/icon';
+import SvgMenu from '@clab-platforms/icon/src/outline/react/Menu';
+import { cn } from '@clab-platforms/utils';
+
+import { SUGANG_PAGES } from '@/widgets/sugang/model';
+import { SugangTableView } from '@/widgets/sugang/ui';
+import { SetStateAction } from 'jotai';
+
+export default function SugangLayout() {
+  const [opened, setOpened] = useState(false);
+
+  return (
+    <div className="flex size-full">
+      <SideBar opened={opened} />
+      <div className="flex size-full flex-col">
+        <Header setOpened={setOpened} />
+        <div className="mb-2 flex size-full flex-col gap-y-2 p-2 px-4">
+          <NavigatorBtn />
+          <SugangTableView tableName="WISH_LIST" />
+          {/*현재는 tableName이 고정되어 있지만, 나중에는 NavigatorBtn을 통해 누른 탭에 따라 Name을 변하게 할 예정입니다.*/}
+          <SugangTableView tableName="REGISTRATION_LIST" />
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function SideBar({ opened }: { opened: boolean }) {
+  return (
+    <div
+      className={cn(
+        'h-full w-44 flex-col bg-yellow-500',
+        opened ? 'flex' : 'hidden',
+      )}
+    >
+      <div className="flex w-full items-center p-[18px] text-white">
+        수강신청시스템
+      </div>
+      <div className="size-full bg-slate-700"></div>
+    </div>
+  );
+}
+
+function Header({
+  setOpened,
+}: {
+  setOpened: Dispatch<SetStateAction<boolean>>;
+}) {
+  return (
+    <div className="flex w-full items-center border-b-2 border-black/20 p-4">
+      <div className="flex items-center gap-x-10">
+        <button onClick={() => setOpened((prev) => !prev)}>
+          <SvgMenu fontSize={20} />
+          {'  '}
+        </button>
+        <KguLogoColor className="h-7 w-24" />
+      </div>
+    </div>
+  );
+}
+
+function NavigatorBtn() {
+  const [selected, setSelected] = useState(0);
+  const sugangPages = SUGANG_PAGES;
+  return (
```

Comment:
코드의 스타일을 한 가지로 유지하는게 좋을 것 같아요.

현재 특정 로직은 `const`와 같은 변수 선언부와 jsx 리턴문간의 개행이 존재하고 특정 로직에는 존재하지 않는데, 저는 개행을 통해서도 코드의 가독성이 높아진다고 생각합니다. 

말을 길게 적었지만 개행으로 적절하게 단락을 분리하게 좋을 것 같다는 이야깁니다 :)

```suggestion
  const [selected, setSelected] = useState(0);
  const sugangPages = SUGANG_PAGES;
  
  return (
```